Client/mods/deathmatch/logic/lua/CLuaFunctionParseHelpers.cpp

```diff
@@ -824,6 +824,27 @@ ADD_ENUM(_D3DFORMAT::D3DFMT_G32R32F, "g32r32f")
 ADD_ENUM(_D3DFORMAT::D3DFMT_A32B32G32R32F, "a32b32g32r32f")
 IMPLEMENT_ENUM_CLASS_END("surface-format")
 
+IMPLEMENT_ENUM_CLASS_BEGIN(eShadowTextureType)
+ADD_ENUM(eShadowTextureType::PLANE, "plane")
+ADD_ENUM(eShadowTextureType::CAR, "car")
+ADD_ENUM(eShadowTextureType::PED, "ped")
+ADD_ENUM(eShadowTextureType::HELI, "heli")
+ADD_ENUM(eShadowTextureType::BIKE, "bike")
+ADD_ENUM(eShadowTextureType::RCBARON, "rcbaron")
+ADD_ENUM(eShadowTextureType::EXPLOSION, "explosion")
+ADD_ENUM(eShadowTextureType::HEADLIGHT1, "headlight1")
+ADD_ENUM(eShadowTextureType::HEADLIGHT2, "headlight2")
+ADD_ENUM(eShadowTextureType::BLOOD, "blood")
+ADD_ENUM(eShadowTextureType::HEADMAN, "headman")
```

Comment:
Isn't this one wrong? `struct RwTexture * gpHandManTex`

Reply:
idk